.circleci/config.yml

```diff
@@ -52,7 +52,8 @@ jobs:
       - run:
           name: check links in changed pages
           working_directory: content
-          command: git diff --name-only -z --diff-filter=AMRCT $(git merge-base HEAD origin/master)..HEAD | bundle exec ./scripts/check-pr-links.rb
+          command:
+            git diff --name-only -z --diff-filter=AMRCT $(git merge-base HEAD origin/master)..HEAD | bundle exec ./scripts/check-pr-links.rb
```

Comment:
YAML is always so disorienting to me, so I'm counting on you to confirm that this syntax has the same effect. 😅

Reply:
i believe so, it was just an autoformatting thing 😄